src/ui-lib/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss

```diff
@@ -14,16 +14,16 @@ body > header {
     justify-content: space-between;
 
     @media (min-width: gravy-breakpoint(medium)) {
-      // Force nav onto same line as logo
-      flex-wrap: nowrap;
+      padding-top: 0;
     }
   }
 
   .logo-main {
     width: 70%; // 2/3 ish of the space
 
     @media (min-width: gravy-breakpoint(medium)) {
-      width: auto;
+      max-width: 250px;
```

Comment:
Perhaps we can bump up the `max-width` a little so that the logos size is closer to what's currently live on the website. I haven't measured it, but to my eyes the logo _looks_ a bit bigger on the website than it does in the styleguide with this PR.

Reply:
So I reduced because right now there's not enough space for the Buildit website nav at ~700px (once the Medium logo is in place). With this new code the nav would wrap below the logo (better than hanging off the screen) but I don't think it's ideal. I can revert it though if you prefer?

Reply:
Ah Ok. That's fine then. Agree that we want to avoid wrapping on the Buildit website if we can.

Reply:
please move these measures into a variable